controllers/k6_start.go

```diff
@@ -15,6 +17,22 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+func isPodReady(pod *v1.Pod) bool {
+	resp, err := http.Get(fmt.Sprintf("%v/v1/status", pod.Status.HostIP))
+
+	if err != nil {
+		return false
+	}
+
+	status, err := strconv.Atoi(resp.Status)
```

Comment:
You can just use `resp.StatusCode`, which is already `int`

Reply:
Thanks! Just fixed it